mmdet/evaluation/metrics/coco_metric.py

```diff
@@ -116,7 +117,7 @@ def __init__(self,
             self._coco_api = None
 
         # handle dataset lazy init
-        self.cat_ids = None
+        self.cat_ids = cat_ids
```

Comment:
cat_ids will be created by the dataset_meta. You don't have to set it manually.

Reply:
https://github.com/open-mmlab/mmdetection/blob/5b0d5b40d5c6cfda906db7464ca22cbd4396728a/mmdet/evaluation/metrics/coco_metric.py#L387-L392

There leaves a custom interface for user definition

Reply:
Okay. If add user-defined cat_ids, please update the docstring and code comments to tell users when and why they need to use it.